sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -19,6 +19,12 @@ internal static class TraceHelper
         private const int Version = 2;
         private const int MaxlinksAllowed = 100;
 
+        internal static readonly IReadOnlyDictionary<TelemetryType, string> s_telemetryItem_Name = new Dictionary<TelemetryType, string>
```

Comment:
a simple if else would be much better than a Dictionary to achieve this.

Reply:
Yes, this list earlier had may values. If else would be better.

Reply:
updated